include/usb/class/usb_hid.h

```diff
@@ -54,6 +54,7 @@ struct usb_hid_descriptor {
 typedef int (*hid_cb_t)(struct usb_setup_packet *setup, s32_t *len,
 			u8_t **data);
 typedef void (*hid_int_ready_callback)(void);
+typedef void (*status_cb_t)(enum usb_dc_status_code status, u8_t *param);
```

Comment:
can memory pointed by param change in the callback? if not pointer should be const

Reply:
I took this from 
```
typedef void (*usb_status_callback)(enum usb_dc_status_code status_code,
				    u8_t *param);
```
in `usb_device.h`

Reply:
I think we can make it const though even in the stack

Reply:
I have fixed this